scripts/main.js

```diff
@@ -0,0 +1,558 @@
+                }
+            });
+        }
+        if (isEng == true) {
+            keyBoard.querySelectorAll(".button").forEach(function (e) {
+                for (let j = 0; j < RU_KEYS_CAPS.length; j++) {
+                    if (EN_KEYS_CAPS[j] == e.getAttribute("data")) {
+                        e.querySelector("p").innerText = e.querySelector("p").innerText.toUpperCase();
+                    }
+                }
+            });
+        }
+        document.querySelector(`.button[data="${20}"]`).classList.add("active-caps");
+        isCaps = true;
+    } else {
+        if (isEng == false) {
+            keyBoard.querySelectorAll(".button").forEach(function (e) {
+                for (let j = 0; j < RU_KEYS_CAPS.length; j++) {
+                    if (RU_KEYS_CAPS[j] == e.getAttribute("data")) {
+                        e.querySelector("p").innerText = e.querySelector("p").innerText.toLowerCase();
+                    }
+                }
+            });
+        }
+        if (isEng == true) {
+            keyBoard.querySelectorAll(".button").forEach(function (e) {
+                for (let j = 0; j < RU_KEYS_CAPS.length; j++) {
+                    if (EN_KEYS_CAPS[j] == e.getAttribute("data")) {
+                        e.querySelector("p").innerText = e.querySelector("p").innerText.toLowerCase();
+                    }
+                }
+            });
+        }
+        document.querySelector(`.button[data="${20}"]`).classList.remove("active-caps");
+        isCaps = false;
+    }
+}
+
+document.addEventListener("mousedown", function (event) {
+    let selectItem = event.toElement;
+    if ((event.target.tagName == "DIV" && event.target.classList.contains("button")) || event.target.tagName == "P") {
+        event.target.tagName == "DIV" ? selectItem.classList.add("active") : selectItem.parentNode.classList.add("active");
+    } else {
+        return;
+    }
+
+    if (event.target.getAttribute("data") == "Space" || selectItem.parentNode.getAttribute("data") == "Space") {
+        setSelectionRange(" ");
+    }
+    if (event.target.getAttribute("data") == 9 || selectItem.parentNode.getAttribute("data") == 9) {
+        setSelectionRange("\t");
+    }
+    if (event.target.getAttribute("data") == 13 || selectItem.parentNode.getAttribute("data") == 13) {
+        setSelectionRange("\n");
+    }
+    if (event.target.getAttribute("data") == 8 || selectItem.parentNode.getAttribute("data") == 8) {
+        let start = textArea.selectionStart,
+            end = textArea.selectionEnd;
+        if (!textArea.setRangeText) { return; }
+        if (start >= end) {
+            if (start <= 0 || !textArea.setSelectionRange) { return; }
+            textArea.setSelectionRange(start - 1, start);
+        }
+        textArea.setRangeText("");
+        textArea.focus();
+    }
+    if (event.target.getAttribute("data") == 46 || selectItem.parentNode.getAttribute("data") == 46) {
+        let start = textArea.selectionStart,
```

Comment:
why not switch case?)

Reply:
> why not switch case?)

Thank you for your comment. Switch cannot have two parameters